src/plugins/Provider/view/Table.js

```diff
@@ -17,15 +19,16 @@ module.exports = (props) => {
   return html`
     <table class="BrowserTable" onscroll=${props.handleScroll}>
       <tbody role="listbox" aria-label="List of files from ${props.title}">
-        ${props.folders.map((folder) => {
+        ${props.folders.map(folder => {
```

Comment:
aww i like the parentheses there. super nit picky tho, i'll adapt either way :D

Reply:
I think it looks cleaner without them, but when multiple args are used, they are needed anyway, so. Hm.

Reply:
its all good haha, lets keep it like this